test/units/parsing/test_splitter.py

```diff
@@ -46,6 +53,18 @@
     (u'a="echo \\"hello world\\"" b=bar',
         [u'a="echo \\"hello world\\""', u'b=bar'],
         {u'a': u'echo "hello world"', u'b': u'bar'}),
+    (u'a="nest\'ed"',
+        [u'a="nest\'ed"'],
+        {u'a': u'nest\'ed'}),
+    (u' ',
+        [u' '],
+        {u'_raw_params': u' '}),
+    (u'\\ ',
+        [u' '],
+        {u'_raw_params': u' '}),
+    (u'a\\=escaped',
+        [u'a\\=escaped'],
+        {u'_raw_params': u'a=escaped'}),
```

Comment:
Although this verifies existing behavior, I'm not entirely sure the output is what it should be. The `\\=` is preserved as-is for `split_args` but is reduced to `=` for `parse_kv`.